README.md

```
@@ -1,207 +1,468 @@
+- `https://www.sundial.exchange` (production with www)
+
+Users browsing the Sundial Exchange website get free API access!
+
+
+For more information about the API: [Complete API documentation and guides](https://github.com/dylanboudro/sundial.exchange)
+<!-- End Summary [summary] -->
+
+<!-- Start Table of Contents [toc] -->
+## Table of Contents
+<!-- $toc-max-depth=2 -->
+* [@starmorph/sundial-exchange-api-typescript](#starmorphsundial-exchange-api-typescript)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)